code/FinanceManager.Components/wwwroot/css/card-info-affordance.css

```
@@ -0,0 +1,144 @@
+    min-height: 28px;
+    padding: 2px 0;
+    border: 0;
+    border-bottom: 1px dotted transparent;
+    background-color: transparent;
+    color: inherit;
+    cursor: help;
+}
+
+.fm-card-info-trigger:hover {
+    color: var(--mud-palette-primary);
+    border-bottom-color: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)